main_test.go

```diff
@@ -0,0 +1,96 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func prepareRespRecorder(s string) *httptest.ResponseRecorder {
+	req := httptest.NewRequest("GET", s, nil)
+
+	responseRecorder := httptest.NewRecorder()
+	handler := http.HandlerFunc(mainHandle)
+	handler.ServeHTTP(responseRecorder, req)
+
+	return responseRecorder
+}
+
+func TestCorrectCountAndCity(t *testing.T) {
+	expectedCities := []string{"Мир кофе", "Сладкоежка"}
```

Comment:
Я бы предложил тогда сразу делать 
```suggestion
	expectedCities := cafeList["moscow"][:2]
```

Reply:
Понял, спасибо за фидбек!